examples/internal_wave.jl

```diff
@@ -161,37 +165,34 @@ nothing #hide
 # We use Makie's `Observable` to animate the data. To dive into how `Observable`s work we
 # refer to [Makie.jl's Documentation](https://makie.juliaplots.org/stable/documentation/nodes/index.html).
 
-iter = Observable(1)
+n = Observable(1)
 
 title = @lift(string("ωt = ",
-              string(round(w_timeseries.times[$iter] * ω, digits=2))))
-
-w = @lift(Array(w_timeseries[:, 1, :, $iter]))
+              string(round(w_timeseries.times[$n] * ω, digits=2))))
 
-# We build the coordinates from the saved `grid`.
+w = @lift(interior(w_timeseries[$n], :, 1, :))
 
-x, y, z = nodes((Center, Center, Center), grid)
 
 # We plot the vertical velocity, ``w``.
 
 w_lim = 1e-8
 
 contourf!(ax, x, z, w; 
-           levels = range(-w_lim, stop=w_lim, length=10),
-           colormap = :balance,
-           colorrange = (-w_lim, w_lim),
-           extendlow = :auto,
-           extendhigh = :auto)
+          levels = range(-w_lim, stop=w_lim, length=10),
+          colormap = :balance,
+          colorrange = (-w_lim, w_lim),
+          extendlow = :auto,
+          extendhigh = :auto)
 
-fig[0, :] = Label(fig, title, textsize=24, tellwidth=false)
+fig[1, :] = Label(fig, title, textsize=24, tellwidth=false)
```

Comment:
```suggestion
fig[1, 1] = Label(fig, title, textsize=24, tellwidth=false)
```

or is it `1:2`? Is there a colorbar?